.github/workflows/pr-notifications.yml

```diff
@@ -48,7 +48,7 @@ jobs:
             --arg desc "> $(echo "$DESC" | sed 's/^/> /')" \
             --arg issues "$(echo "$ISSUES" | sed 's/^/> /')" \
             '{
-              channel: "github-test",
+              channel: "acf-ohs-ttahub--contractor-customer-team",
```

Comment:
Have no idea if something like this may change in the future, but could stuff the channel name into a GitHub Action variable to separate config from code.